### DISCUSSION TITLE
Retirement of dakota-users listserv

### DISCUSSION THREAD

**Discussion by jadamstephens** [snl-dakota/dakota#9] [Announcements]
The Dakota team is excited to announce that user support is transitioning to [GitHub Discussions](https://github.com/orgs/snl-dakota/discussions). The dakota-users listserv will be retired.

Although dakota-users has served our community well over the years, GitHub Discussions has numerous features that we believe warrant this change. Some of these include:

* Searchable history
* Threaded discussions
* markdown formatting
* LaTeX style mathematical expressions
* Voting, best answer selection

To use GitHub Discussions, you'll first need to sign up for a GitHub account if you don't already have one. Once you have an account, you can visit our[ Discussions ](https://github.com/orgs/snl-dakota/discussions)page and create a New Discussion or participate in discussions that others have created. If you'd like to receive notifications about posts in Discussions, go to the [main dakota repository page](https://github.com/snl-dakota/dakota) and click Watch (upper right corner)→Custom→Discussions→Apply.

Our Discussions page is active now. The dakota-users listserv will continue to operate until February 15, 2023. We will send a couple of reminders before then.

**Comment by HectorLestani** [comment]
This is really good news.
Congratulations to all the team behind this!
Cheers.
